test/inductor/test_triton_kernels.py

```diff
@@ -69,6 +69,36 @@ def f(x):
         # No need to assert anything, the goal is to make sure dynamo does
         # not crash
 
+    @requires_gpu
+    def test_data_ptr_packing(self):
+        @triton.jit
+        def add_kernel(xyz, BLOCK_SIZE: "tl.constexpr", n_elements):
+            pid = tl.program_id(axis=0)
+            block_start = pid * BLOCK_SIZE
+            offsets = block_start + tl.arange(0, BLOCK_SIZE)
+            mask = offsets < n_elements
+            x_ptr = tl.load(xyz + 0).to(tl.pointer_type(tl.float32))
+            y_ptr = tl.load(xyz + 1).to(tl.pointer_type(tl.float32))
+            z_ptr = tl.load(xyz + 2).to(tl.pointer_type(tl.float32))
+
+            x = tl.load(x_ptr + offsets, mask=mask)
+            y = tl.load(y_ptr + offsets, mask=mask)
+            output = x + y
+            tl.store(z_ptr + offsets, output, mask=mask)
+
+        # @torch.compile(fullgraph=True)
```

Comment:
test still failing, we need torch.tensor to be traceable too :/ (but this is unrelated to the TMA work)